src/Components/Details.css

```diff
@@ -0,0 +1,25 @@
+#details {
+    color: white;
+    font-size: 15px;
+    margin: 0 auto;
+    width: 400px;
+    height: 150px;
+    text-align: center;
+    border: 1px solid white;
+    padding: 10px;
+    margin-top: 100px;
+}
+
+#homeBtn {
```

Comment:
Two things here:

1. I wouldn't use Ids (`#`), since they are supposed to be unique, and that's very hard to guaranee
2. Since we want to isolate everything as part of the `Details` component, could you please isolate your CSS changes as well? e.g. rather than having `.homBtn`, I'd rather prefer `.details .homeBtn`, so if you defines another `.homeBtn` anywhere else your changes wouldn't affect the latter.